src/plugins/event-plugins/JsErrorPlugin.ts

```diff
@@ -6,14 +6,17 @@ export const JS_ERROR_EVENT_PLUGIN_ID = 'com.amazonaws.rum.js-error';
 
 export type PartialJsErrorPluginConfig = {
     stackTraceLength?: number;
+    errorsToExclude?: RegExp[];
```

Comment:
issue: A callback function would give application maintainers more control over which errors should be included/excluded. What if we make this a function whose behavior is similar to [Array.filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)?

Specifically, the function would take as input an error and return `true` if the error should be *included* in the RUM events.

Reply:
Vote up for this one. BTW I also wondered some other topics: why supporting `string` is not needed? In my opinion, most web apps are running in different browser environments which means some native error messages from browsers will be in different formats.

Reply:
Great suggestion, I have updated the PR so that customers will pass in a "filter" like function that will `true` when the error should be ignored (thought this is more intuitive - more than welcome to change if anyone disagrees).